packages/gcf-utils/src/gcf-utils.ts

```diff
@@ -157,10 +157,11 @@ export interface CronPayload {
  * Type of function execution trigger
  */
 export enum TriggerType {
-  GITHUB = 'GITHUB_WEBHOOK',
-  SCHEDULER = 'SCHEDULER',
-  TASK = 'TASK',
-  UNKNOWN = 'UNKNOWN',
+  GITHUB = 'GitHub Webhook',
```

Comment:
nit: but  curious why we also changed the value of this enum in this PR; not a blocker just a question.

Reply:
just for better logs readability